src/js-utils/Glennsl__BsBenchmarkJs__JsUtils.re

```diff
@@ -0,0 +1,18 @@
+module Obj = {
+
+  let immutableUpdate = (~props, obj) => Js.Obj.(()->empty->assign(obj)->assign(props));
+  
+  let shallowCopy = [%bs.raw {|function(obj) {
+      var _obj = {};
+      var keys = Object.keys(obj);
+      var length = keys.length;
+      for (var i = 0; i < length; ++i) {
+        var key = keys[i];
+        _obj[key] = obj[key];
+      };
+      return _obj;
+    }|}];
+
+  let empty = [%bs.raw {|function() { return {}; }|}];
```

Comment:
This already exists as `Js.Obj.empty`

Reply:
Hmm, I think the abstract `options` type I used was incompatible with the `Js.t({..})` type produced by `Js.Obj.empty`, but in the end, I removed this code and a lot of other stuff related to `options` and `suiteOptions`, which simplified things a lot.

In the end, the user really just needs to be able to pass options to the constructor. Reading/mutating/copying those objects seems like a niche use case to me...